PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/CacheManager.java

```diff
@@ -0,0 +1,36 @@
+package org.prebid.mobile;
+
+import android.os.Handler;
+import android.text.TextUtils;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+public class CacheManager {
+    private static HashMap<String, String> savedValues = new HashMap<>();
+    private static Handler handler = new Handler();
```

Comment:
Add Expiry logic here, that native ads expires.

Reply:
Done.